ci/repo_cleanups/remove_stale_rpm.sh

```diff
@@ -0,0 +1,231 @@
+#!/bin/bash 
+
+
+function run_git {
+#   git for-each-ref --sort=-committerdate refs/remotes --format='%(HEAD)%(refname:short), %(committerdate:relative)'
+    git for-each-ref --sort=-committerdate refs/remotes --format='%(refname:short), %(committerdate:relative)'
+}
+
+
+function write_script {
+
+    var=$(cat <<"EOF"
+
+    declare -A  git_branches    ## Store the list of git branches in a hash
```

Comment:
I wonder if there is a way to do it without using hashes. They require bash version 4 and it's a complicated data structure which is not what bash is generally good at.

I haven't understood the script enough yet to provide a good alternative but I'll keep reading a bit more.